### ISSUE TITLE
not able to pass parser options

### ISSUE THREAD

**Issue by timothyswt** [intuit/node-pom-parser#12]
the options are hardcoded and cannot be overriden. You should retain key camelcasing by default.

**Comment by foobar98** [comment]
@nsinghal12 I don't see the options being hardcoded anywhere. Not valid anymore I guess.

**Comment by nsinghal12** [comment]
@foobar98 Parser options are hardcoded at line 10. This is a valid issue.

**Comment by foobar98** [comment]
Okay, I thought he meant the main parse options. What should be the ideal solution in this case then - clients can pass xmljsOptions as part of the `parse` options? And if nothing is provided we default to our hardcoded values?

**Comment by nsinghal12** [comment]
@foobar98 Yes, we should add these options to the first argument in `parse` call and fallback to defaults when not provided.

**Comment by nsinghal12** [comment]
This issue is available to be picked up for hacktoberfest. Please comment here if anyone wants to pick this one.

**Comment by foobar98** [comment]
Willing to take this if nobody else is.
